### DISCUSSION TITLE
Slug getting index on google

### DISCUSSION THREAD

**Discussion by Onibenjo** [vercel/next.js#29495] [Help] [answered]
### What version of Next.js are you using?

11.0.1

### What version of Node.js are you using?

16.9.1

### What browser are you using?

Edge

### What operating system are you using?

macOS

### How are you deploying your application?

next build

### Describe the Bug

Slug getting indexed
![image](https://user-images.githubusercontent.com/40042573/135266823-31b818ea-15b8-410f-b3e2-8f436adc50b9.png)

### Expected Behavior

Generated pages should be indexed

### To Reproduce

google 'daba blog'

**Comment by ijjk** [comment, marked as answer]
Hi, it looks like you rendered a link with `/blog/[slug]` on one of those pages instead of using the actual `href` for the page.

**Reply by Onibenjo** [reply]
Oh, will check it out, thanks

**Reply by Onibenjo** [reply]
dont have any link being rendered with `/blog/[slug]`

**Reply by ijjk** [reply]
Are you checking in the initial prerendered HTML or the after hydration on the client? This is most likely happening during prerendering on the server. You can do a search in `.next/server/pages/**/*.html` to see if `/blog/[slug]` is present.

**Reply by Onibenjo** [reply]
alright

**Reply by Onibenjo** [reply]
I think i have gotten the root of the problem, I was using `router.route` instead of `router.asPath` in my seo
